idle-rpg/bots/data/commands.js

```diff
@@ -142,6 +142,32 @@ const commands = [
     }
   },
 
+ lore = {
+    command: '!lore',
+    operatorOnly: false,
+    channelOnlyId: commandChannel,
+    function: (game, message, discordBot) => {
+      let requestedMap = message.content.split(' ')[1];
+      let mapNameArray = [];
+      maps.forEach ((mapObject) => {
+        mapNameArray.push(mapObject.name)
+      });
+      const mapIndexLookedFor = mapNameArray.indexOf(requestedMap);
+      
+      if (mapIndexLookedFor === -1) {
+        console.log('Are you sure you have the correct spelling? Captilization counts!');  
+        //message.author.send(`\`\`\`Are you sure you have the correct spelling? Captilization counts!\`\`\``);
```

Comment:
The returns go here so the bottom code does not get executed when there is no map that was searched for